ktor-server/ktor-server-tomcat/jvm/src/io/ktor/server/tomcat/TomcatApplicationEngine.kt

```diff
@@ -72,12 +72,17 @@ public class TomcatApplicationEngine(environment: ApplicationEngineEnvironment,
                         if (ktorConnector.keyStorePath == null) {
                             throw IllegalArgumentException("Tomcat requires keyStorePath")
                         }
+                        if (ktorConnector.trustStorePath != null) {
```

Comment:
We probably should fail if only `trustStore` is specified with no `trustStorePath`, right?

Reply:
For Tomcat only:
Usecase 1: Truststore null, truststorePath null => client cert Auth not wanted
Usecase 2: Truststore set, trustStorePath null => client cert wanted, but no correct config. Currently there is NO error shown to the user. Do you want an error here?
Usecase 3: truststore null, truststorePath set => client cert wanted and correct config (for tomcat only)
Usecase 4: truststore set, truststorePath set => client cert wanted and correct config

Reply:
I think yes, we shouldn't silently ignore config, especially auth

Reply:
Exception added :)